app/models/timelog.rb

```diff
@@ -5,7 +5,8 @@ class Timelog < ApplicationRecord
   validates :user_id, presence: true
   validates :start_time, presence: true
   validates :duration, presence: true, numericality: { greater_than: 0 }
-  validates :trello_card, presence: true
+  validates :task_description, presence: true, unless: ->(timelog){timelog.trello_card.present?}
+  validates :trello_card, presence: true, unless: ->(timelog){timelog.task_description.present?}
```

Comment:
spaces between brackets

Reply:
looks like I do need to consider moving from RubyMine :)